lib/katello/tasks/upgrades/2.4/import_package_groups.rake

```diff
@@ -0,0 +1,12 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '2.3' do
+      task :import_package_groups => ["environment"]  do
+
+        User.current = User.anonymous_api_admin
+        puts _("Importing Package Groups")
+        Katello::PackageGroup.import_all
```

Comment:
@jlsherrill Since this is doing the same thing as reindex, should we just default to calling reindex instead on an upgrade? I realize that is a much heavier weight operation in general (since it would apply it across the board).